Added bandwidth history for usage on cloud (#5061)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/usage.md

## Usage overview

> The overview shows the bandwidth of the project this month, the media library size, and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.

Reviewer: ```suggestion
The overview shows the bandwidth of the project for the current month, the media storage size, and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.
```

> The usage overview shows the bandwidth usage of the project this month, the size of the media library and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.

Author: ```suggestion
The overview shows the bandwidth of the project this month, the media library size, and the number of custom domains added to the project. It is also possible to see the bandwidth history for the previous six months.
```

> In this overview you will also find the usage limitations for your Umbraco Cloud project as well as the plan that the project is on.

Reviewer: ```suggestion
In this overview, you will find the usage limitations for your Umbraco Cloud project as well as the plan that the project is on.
```
